ui/webui/src/components/installation/ReviewConfiguration.jsx

```diff
@@ -14,16 +14,143 @@
+                    setSelectedDisks(res);
+                    // get detailed data for the selected disks
+                    res.forEach(disk => {
+                        console.info("processing disk");
+                        console.info(disk);
+                        getDeviceData({ disk })
+                                .then(res => {
+                                    console.info("disk device data");
+                                    console.info(res);
+                                    setDeviceData(d => ({ ...d, [disk]: res[0] }));
+                                }, console.error);
+                    });
+                }, console.error);
+    }, []);
+
+    // handle case of disks not (yet) loaded
+    if (!selectedDisks) {
+        return null;
+    }
+
+    return (
+        <>
+            <Title headingLevel="h2" size="2xl">
+                {_("Review and install")}
+            </Title>
+            <DescriptionList>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>
+                        {_("Language")}
+                    </DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {systemLanguage}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+            <Title headingLevel="h3" size="xl">
+                {_("Installation destination")}
+            </Title>
+            <DataList>
+                {selectedDisks.map(selectedDisk => (
+                    <DataListItem key={selectedDisk}>
+                        <DataListItemRow>
+                            <DataListItemCells
+                              dataListCells={[
+                                  <DataListCell key={selectedDisk} id={"local-disks-item-" + selectedDisk}>
+                                      {_("Local standard disk")}
+                                  </DataListCell>,
+                                  <DataListCell key={"description-" + selectedDisk}>
+                                      {deviceData && deviceData[selectedDisk] && deviceData[selectedDisk].description.v + " (" + selectedDisks + ")"}
+                                  </DataListCell>,
+                                  <DataListCell key={"size-" + selectedDisk}>
+                                      {cockpit.format_bytes(deviceData && deviceData[selectedDisk] && deviceData[selectedDisk].size.v) + " " + _("total")}
+                                  </DataListCell>
+                              ]}
+                            />
+                        </DataListItemRow>
+                    </DataListItem>
+                ))}
+            </DataList>
+        </>
+    );
+};
+
+export const ReviewConfigurationFooter = ({ isEraseChecked, setIsEraseChecked }) => {
     return (
-        <Form isHorizontal>
-            Imagine configured values for review here.
-        </Form>
+        <>
```

Comment:
Unnecessary empty wrapper. Flex can be wrapper by itself.

Reply:
I noticed it being used so I assumed there was some good reason for that. :)

Reply:
When you return JSX you need to always return one element as wrapper. If you want to return two buttons for example you can do:

``` jsx
<> 
  <Button> 1 </Button>
  <Button> 2 </Button>
</>
```

If you already use a component as wrapper then you don't need this 'empty' wrapper. So this is valid code.

``` jsx
<Flex>
  <Button> 1 </Button>
  <Button> 2 </Button>
</Flex>
```

Read more about it here:
https://reactjs.org/docs/fragments.html

Note: <> is the same like React.Fragment

Reply:
Oh, interesting, so essentially short-hand for Fragment. Yeah, I remember from other framework that you need to have on top level thing/parent as well but what confused me that you can just use what essentially looks like empty brackets. :)